README.md

```diff
@@ -0,0 +1,55 @@
+# Now build awesome UIs  with Mlan ui.
+
+### Make your web pages beautiful and responsive using Mlan UI, without much hassle in css
+---
+**How to use MLAN UI?** 
+- To use MLAN UI in your project you need to copy and paste the below link, and add this inside the <head> tag of your HTML file OR you can import the code into your CSS file.
+
+> `<link rel="stylesheet" href="https://mlanui.netlify.app/library/style.css" />`
+
+> `@import url("https://mlanui.netlify.app/library/style.css")`
```

Comment:
try wrapping the code in triple backticks then you'll get a better looking code block with a copy button.
example - 
![codereview](https://user-images.githubusercontent.com/95222845/154858612-8b41cda9-76c0-4966-a4db-34dde7196850.png)

output
```html
<link rel="stylesheet" href="https://mlanui.netlify.app/library/style.css" />
```
```css
@import url("https://mlanui.netlify.app/library/style.css")
```

Reply:
I will warp the code , thanks sudip